pkg/monitoring/configmaps.go

```diff
@@ -24,12 +25,19 @@ func LoggingConfigMaps(managedClusterName string) []*corev1.ConfigMap {
 	if err != nil {
 		zap.S().Debugf("New logging config map %s is giving error %s", filebeatindexconfig.Name, err)
 	}
-	filebeatconfig, err := createLoggingConfigMap(constants.LoggingNamespace, "filebeat-config", "filebeat.yml", FilebeatConfigData, filebeatLabels)
+
+	filebeatConfigData := FilebeatConfigDataDocker
+	filebeatInputData := FilebeatInputDataDocker
+	if strings.HasPrefix(containerRuntime, "containerd://") {
```

Comment:
Maybe make `"containerd://"` a constant.

Reply:
Yeah, I will add a constant. I was going to encapsulate this in a struct but decided against it for simplicity.

Reply:
Addressed in commit e0cb7e9523fa6287222cc134a03a69a0cfd26fa2